packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1781,6 +1781,18 @@ export default class NewVm extends BaseComponent {
             <SectionContent>
               <Item label={_('secureBoot')}>
                 <Toggle onChange={this._toggleState('secureBoot')} value={secureBoot} />
+                {secureBoot && !pool.secureBootSetup === '' && (
```

Comment:
As @pdonias said [here](https://github.com/vatesfr/xen-orchestra/pull/7562#discussion_r1568389150), `pool.secureBootSetup` should be a boolean.
```suggestion
                {secureBoot && !pool.secureBootSetup && (
```